infra/terraform/base_stacks_2/README.md

```diff
@@ -0,0 +1,5 @@
+# Before applying changes
+
+For initial configuration please refer to the README file located in ./infra/terraform
+
+This base_stacks_2 directory is only to be deployed after the deployment of the original base_stacks as dns_records.tf is dependent on dns.tf being deployed first.
```

Comment:
Please add in details about the dependency on the configuration in the devplatform account which hosts the base route53 record. 

Having a dependency on the `dns` stack is not sufficient to explain why this is its own terraform configuration.